index.html

```diff
@@ -1331,6 +1331,27 @@
 A more accurate implementation should in principle account for these effects.
 
 
+White furnace testing
+-------------------------------------
+
+The ground truth appearance should perfectly preserve energy in various configurations, in which case the material will satisfy a "white furnace test", i.e.  the object should disappear when illuminated by uniform background light, and rendered in the limit of a large number of bounces. We list here for convenience the configurations in which such a white furnace should pass, all of which would be useful to check as part of a unit testing framework:
+
+   - A fully metallic base (**`base_metalness`**=1) with white **`base_color`** and white **`specular_color`**.
+   - A fully dielectric base (**`base_metalness`**=0), with white **`specular_color`**=0, including:
```

Comment:
What does "white **`specular_color`**=0" mean? Did you intend to write "white **`specular_color`**=(1,1,1)" ?